cypress.env.json

```diff
@@ -5,5 +5,8 @@
   "Button": "http://localhost:9001/?path=/story/button--default",
   "Button Toggle":"http://localhost:9001/?path=/story/button-toggle--default",
   "Button Toggle Group": "http://localhost:9001/?path=/story/button-toggle-group--default",
-  "Dialog Full Screen": "http://localhost:9001/?path=/story/dialog-full-screen--default"
+  "Dialog Full Screen": "http://localhost:9001/?path=/story/dialog-full-screen--default",
```

Comment:
The contents of this file have changed so that we're only storing the base URL here. You might want to communicate with Marcin about the changes he's made here.

Reply:
Yes, U're right. After discuss with @marcinstachowicz, I need to change it.

Reply:
fixed